draft-ietf-taps-arch.md

```diff
@@ -228,7 +228,7 @@ The normative requirements described here allow Transport Services APIs and Impl
 
 ## Provide Common APIs for Common Features
 
-Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of Transport Services API calls. As a baseline, any Transport Services API MUST allow access to the minimal set of features offered by transport protocols {{?RFC8923}}.
+Any functionality that is common across multiple transport protocols SHOULD be made accessible through a unified set of Transport Services API calls. As a baseline, any Transport Services API SHOULD allow access to the minimal set of features offered by transport protocols {{?RFC8923}}.
```

Comment:
I think worrying about this could be picky, but it would be good to be consistent.

To me - applications in general really ought NOT to use this. The transport makes this decision, so this really should be "management" to identify persistent problems. If that is so, it's rather like 6.2.17.  "Notification of ICMP soft error message arrival...", in fact why couldn't it be actually caught by the same call?  Making that call "Notification of soft error" seems much more appropriate and extensible. It might allow a TAPS system to include soft errors notified also from subnetwork control protocols, such as tunnel endpoints or other non-data plane signals?

Reply:
YES, I agree!  I guess it doesn't even need a technical change, this notification IS categorized as a "soft error" in RFC 1122, section 4.2.4.1: "These error reports generally reflect soft errors that can be ignored without harm by many applications."

So I think the only thing that needs to be done is to update the minset appendix to reflect this instead of saying "we don't support this". Right?

Reply:
OK:
@abrunstrom: Do we need to discuss this PR? 
@mwelzl I suggest a separate PR to do this.